document/src/main/resources/config/liquibase/changelog/20220215-create-document-definition-role-changelog.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
```

Comment:
Existing project that migrate to this version, will they keep seeing their cases?

Do we need to add a migration script that will make sure that all users of existing projects will keep seeing all cases?

Reply:
They will always see them as an administrator.
But no by default all the cases will be disabled for all roles. So the admin must set the correct roles before they will be visible for a ROLE_USER

Reply:
I added a migration script for existing installations, where ROLE_USER is assigned to the document definitions.
Any new definitions added during or after the upgrade will not have any role assigned by default.